### ISSUE TITLE
Publish the merkle root of the executed orders

### ISSUE THREAD

**Issue by clangenb** [olisystems/worker#8]
After #6 and #7 have been done, we can also publish the merkle root on the integritee-node.  The preliminaries have been integrated in the pallets with https://github.com/integritee-network/pallets/pull/151. However, this needs minor integration work on the worker side before this is tackled here: https://github.com/integritee-network/worker/issues/1171.

The corresponding PR in the worker is open it also shows how to create the `publish_hash` extrinsic that is sent to the integritee-node (in another context, it is not a merkle root that is published). However, it depends on another PR, which needs to be merged. This is the PR: https://github.com/integritee-network/worker/pull/1174. It should be merged soon.

**Comment by clangenb** [comment]
Done
